growth/too/views.py

```diff
@@ -769,6 +769,29 @@ def validate_filters(self, field):
                 'Some filters are not available for this telescope: ' +
                 ', '.join(unavailable_filters))
 
+    def validate_field_ids(self, field):
+
+        telescope = self.telescope.data
+        filters = self.filters.data.split(",")
+        field_ids = [int(x) for x in self.field_ids.data.split(",")]
+        doRef = bool(self.references.data)
+
+        bands = {'g': 1, 'r': 2, 'i': 3, 'z': 4, 'J': 5}
```

Comment:
Don't we have this mapping dictionary hardcoded in several places already?

Reply:
I moved it now to the top to define it for both functions.

Reply:
Yes, but isn't it also in models.py?